docs/source/deployment/lambda.md

```diff
@@ -58,6 +59,226 @@ exports.graphqlHandler = server.createHandler();
 
 Finally, **pay close attention to the last line**. This creates an export named `graphqlHandler` with a Lambda function handler.
 
+## Deploying with the AWS Serverless Application Model (SAM)
+
+The [AWS Serverless Application Model (AWS SAM)](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/what-is-sam.html) is an open-source framework that you can use to build serverless applications on AWS.
+
+### Configuring your SAM template.yaml
+
+An AWS SAM template file closely follows the format of an AWS CloudFormation template file, which is described in [Template anatomy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/template-anatomy.html) in the AWS CloudFormation User Guide.
+
+For the sake of this example, the following file can just be copied and pasted into the root of your project.
+
+```
+# template.yaml
+
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+
+Resources:
+  GraphQLFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: graphql/
+      Handler: app.graphqlHandler
+      Timeout: 5 #
+      MemorySize: 256
+      Runtime: nodejs14.x
+      Events:
+        AutomaticPersistedQueries:
+          Type: Api
+          Properties:
+            Path: /
+            Method: get
+        GraphQL:
+          Type: Api
+          Properties:
+            Path: /
+            Method: post
+        ApolloSandbox:
+          Type: Api
+          Properties:
+            Path: /
+            Method: options
+```
+
+You will need to ensure `options` are exposed to use Apollo Explorer and Apollo Sandbox.
+
+*Note: The example above is using v1 of AWS API Gateway which is denoted by the `Type: Api`, but v2 (`Type: HttpApi`) is supported as well. The `apollo-server-lambda` [repository currently has tests that cover both v1 and v2 API Gateway events and ALB events](https://github.com/apollographql/apollo-server/blob/main/packages/apollo-server-lambda/src/__tests__/lambdaApollo.test.ts). More info about API Event Source can be found [here](https://github.com/aws/serverless-application-model/blob/master/versions/2016-10-31.md#api)*
```

Comment:
Why are we choosing an older thing as our default instead of a newer thing?

Also missing period at end of line. Also we usually format notes as block quotes (start with `> `) instead of italics.

Reply:
This one is definitely a little confusing with how Amazon does this. Honestly I defaulted to v1 because of a discussion I had with some AWS Solutions Architect about this work. I was told a lot of people still defaulted to v1 because it had more features. 

We could point them at [the AWS docs feature matrix](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-vs-rest.html) that might be more helpful. In that matrix at the top, AWS calls out the previous REST API having more features:
>Previous-generation REST APIs currently offer more features